src/App.css

```diff
@@ -13,10 +13,12 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
 }
 
 .book-container {
   display: flex;
```

Comment:
Great job @Simpleshaikh1 🤝

- [ ] As stated in the project requirements, kindly ensure you do not repeat the same rule. You may create a single class for the `display: flex`, and the `align-items: center` rules, and then reuse these rules in your JSX code. Kindly use this [article](https://speckyboy.com/good-bad-css-practices/) for more guidance.